configuration/concepts/concepts.csv

```diff
@@ -312,11 +312,16 @@ a5890d50-681b-4ff6-a36b-f0cb6ee28c04,,,At bedtime,នៅពេលចូលគេ
 88e55059-0ba4-11eb-a3aa-0242ac140008,,SNOMED:258705008,Week(s),សប្ដាហ៍,Week(s),សប្ដាហ៍,,,Misc,N/A,,,,,,,,,,,
 88e5cc3c-0ba4-11eb-a3aa-0242ac140008,,SNOMED:258706009,Month(s),ខែ,Month(s),ខែ,,,Misc,N/A,,,,,,,,,,,
 9d735b63-3f10-11e4-adec-0800271c1b75,,,Duration Units,ឯកតានៃថិរវេលា,Duration Units,ឯកតានៃថិរវេលា,,,ConvSet,N/A,,,Minute(s); Hour(s); Day(s); Week(s); Month(s),,,,,,,,
-9d77abd7-3f10-11e4-adec-0800271c1b75,,,QD - 1X /day,QD - 1X /ថ្ងៃ,QD - 1X /day,QD - 1X /ថ្ងៃ,,,Misc,N/A,,,,,,,,,,,
-9d7c81f9-3f10-11e4-adec-0800271c1b75,,,BID - 2X /day,BID - 2X /ថ្ងៃ,BID - 2X /day,BID - 2X /ថ្ងៃ,,,Misc,N/A,,,,,,,,,,,
-9d7d753b-3f10-11e4-adec-0800271c1b75,,,TID - 3X /day,TID - 3X /ថ្ងៃ,TID - 3X /day,TID - 3X /ថ្ងៃ,,,Misc,N/A,,,,,,,,,,,
-9d83d5f2-3f10-11e4-adec-0800271c1b75,,,QID - 4X /day,QID - 4X /ថ្ងៃ,QID - 4X /day,QID - 4X /ថ្ងៃ,,,Misc,N/A,,,,,,,,,,,
-9d8e6918-3f10-11e4-adec-0800271c1b75,,,STAT - Immediately,STAT - ភ្លាម,STAT - Immediately,STAT - ភ្លាម,,,Misc,N/A,,,,,,,,,,,
+9d77abd7-3f10-11e4-adec-0800271c1b75,True,,,,,,,,,,,,,,,,,,,,
+9d7c81f9-3f10-11e4-adec-0800271c1b75,True,,,,,,,,,,,,,,,,,,,,
+9d7d753b-3f10-11e4-adec-0800271c1b75,True,,,,,,,,,,,,,,,,,,,,
+9d83d5f2-3f10-11e4-adec-0800271c1b75,True,,,,,,,,,,,,,,,,,,,,
+9d8e6918-3f10-11e4-adec-0800271c1b75,True,,,,,,,,,,,,,,,,,,,,

```

Comment:
Let's move all retiring entities in their own retirement CSV file (for clarity and separation of concerns) and let's use the UUID as a filler value for required fields (hopefully only en FSN?). This will prevent build time validation errors for the CI through the Maven plugin.